.travis.yml

```diff
@@ -10,11 +10,11 @@ addons:
     organization: "lequal"
 
 jdk:
-  - oraclejdk8
```

Comment:
Could you please revert your modifications on the `.travis.yml` file ?
This issue regarding Java8 was resolved in the dev branch, and we'll fix our sonarcloud key issue on this branch too.